dev/NumberBox/NumberBox.xaml

```diff
@@ -99,6 +99,8 @@
                             FontWeight="Normal"
                             Foreground="{ThemeResource TextControlHeaderForeground}"
                             Margin="{ThemeResource TextBoxTopHeaderMargin}"
+                            FontSize="{TemplateBinding FontSize}"
+                            FontFamily="{TemplateBinding FontFamily}"
```

Comment:
Why isn't FontWeight needed here?

Reply:
We already have added "FontWeight=Normal", which I assumed was added for a reason (same why we enforce certain font sizes on other controls and their subcontrols).